tests/types_test.py

```diff
@@ -119,28 +119,28 @@ def test_parse_measure_command_response_including_units() -> None:
                     "unit": "%",
                 },
                 "latency_total": {
-                    "unit": "ms",
+                    "unit": "milliseconds",
```

Comment:
This one is weird to me too when you consider how thoroughly we have wired against Golang style `Duration` strings all over the codebase and into the config.

If we need/want expanded long-form units we can probably do something clever and sexy with an enum subclass

Reply:
I actually don't buy the notion that people won't interpret "ms" at a glance properly.￼ It's pretty standard across Posix-everything.